cmake/cmake_uninstall.cmake.in

```diff
@@ -0,0 +1,22 @@
+# From: https://cmake.org/Wiki/CMake_FAQ
```

Comment:
the page has moved to new location:
https://gitlab.kitware.com/cmake/community/wikis/FAQ#installation-questions
consider changing